NuGet.config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <clear />
     <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
+    <add key="dotnet-iot" value="https://dotnetfeed.blob.core.windows.net/dotnet-iot/index.json" />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="myget.org" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />

```

Comment:
Do you still need the myget dependency?

Reply:
I don't think so, but forgot to remove it. I'll take care of that in a subsequent PR. Thanks for the review!